### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by jafors**
Maybe wait for reraised errors - I'll commit here if anything changes in test behaviour.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,129 +1,4 @@
...
-.mypy_cache/
-.dmypy.json
-dmypy.json
-
-# Pyre type checker
-.pyre/
+poetry.lock
+vembrane.egg-info/*
+tests/__pycache__/*
+vembrane/__pycache__/*
```
Comment (tedil) [inline .gitignore:4]: I'd like to keep the .gitignore we already have (and check if it covers the cases you provided here)

Comment (jafors) [reply .gitignore:4]: Sorry, somehow there went something wrong. We can remove this here.

Comment (tedil) [reply .gitignore:4]: Do you mean the .gitignore or this PR?

Comment (jafors) [reply .gitignore]: I updated the gitignore with some stuff that wasn't catched last time. Everything should be fine here now

Comment (tedil) [reply .gitignore]: Thank you! <3

**Pull request merged**